src/pages/Goods_view/GoodsView.js

```diff
@@ -1,5 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import MainOptionsBox from './Main_options_box/MainOptionsBox';
+import SubOptionsBox from './Sub_options_box/SubOptionsBox';
+import './GoodsView.scss';
+import './mock.json';
+import { API } from '../../config';
+
+const OPTION_OBJECT = {
+  '껍질 제거': 1000,
+  시즈닝: 2000,
+  '양념소스 추가': 1000,
+};
+function priceToString(price) {
+  return price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
 function GoodsView() {
-  return <></>;
+  const [isShowOptionBox, setIsShowOptionBox] = useState(false);
+  const [selectedOptions, setSelectedOptions] = useState();
+  const [resultPrice, setResultPrice] = useState(0);
+  const [productData, setProductData] = useState(null);
+  // const intNumber = parseInt(productData.price);
+  // const params = useParams();
+  useEffect(() => {
+    // fetch('http://localhost:3000/data/mock.json')
+    // fetch('http://10.58.7.0:8000/products/1')
+    fetch(API.goodsview)
+      .then(res => res.json())
+      // .then(data => console.log(data));
+      .then(data => setProductData(data.data));
+  }, []);
+
+  return (
+    productData !== null && (
+      <div>
+        <div className="product_detail_container">
+          <div className="inner_wrap">
+            <div className="detail_top">
+              <div className="detail_left">
+                <div className="detail_image_container">
+                  <div className="detail_image_wrapper">
+                    <img src={productData.image[0]} />
+                    <img src={productData.image[1]} />
+                    <img src={productData.image[2]} />
+                  </div>
+                </div>
+              </div>
+              <div className="detail_right">
+                <div className="detail_information">
+                  <div className="detail_information_title">
+                    {`${productData.name}`}
```

Comment:
templite literals로 한번 더 감싸주신 이유가 있으실까요? 아래와 같이 작성해도 될 것 같네요!
```suggestion
                    { productData.name }
```

Reply:
아 휴먼에러입니다....